### ISSUE TITLE
[Reset Password Form] Cannot tab to "Show my typing" toggle

### ISSUE THREAD

**Issue by carodew** [18F/usfwds#415]
**Steps to recreate**

On the reset password form, set focus on the first form field and tab through the form. The tab order skips the "Show my typing" control. 

![image](http://g.recordit.co/g6qHq0e21A.gif)

**Expected behavior**
Tab order should be: 
1. New password
2. Retype password
3. Show my typing
4. Reset password button

This allows keyboard users ability to access this functionality. 

**Tested in**
Macbook Pro
Yosemite 10.10.5
Firefox 39.0.3
staging

**Comment by maya** [comment]
This is a browser setting that users have to set in order to access text links via tab. And is only for Firefox in which it's a setting. In Safari, you must use option + tab to access this. See: http://www.456bereastreet.com/archive/200906/enabling_keyboard_navigation_in_mac_os_x_web_browsers/ 

This works in Chrome which has it enabled by default:

<img width="411" alt="screen shot 2015-09-09 at 11 15 30 am" src="https://cloud.githubusercontent.com/assets/5249443/9770338/8ff53884-56e4-11e5-8693-4a7b400dc3e6.png">

**Comment by carodew** [comment]
Discussed in bug triage and closing for the reasons stated above – 
*it's a setting in FF
*users of FF who need this enabled probably already have
*overriding this might get wonky.
